scss/application.scss

```diff
@@ -31,6 +31,43 @@ body {
   padding: 50px;
 }
 
+.page-footer {
+  height: $footer-height;
+  background-color: $blue;
+}
+
+.contact {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  color: $white;
+  font-size: $font-big;
+
+  &-input {
+    width: 400px;
```

Comment:
You should set a `max-width` and a `width: 100%;` so that way the layout won't break in responsive mode :smile: